utilities.lib.nut

```diff
@@ -157,28 +183,20 @@ utilities._isLeapYear <- function(y) {
 // **********         Public           **********
 utilities.getNewUUID <- function(cb = null) {
     if (cb == null) {
-        server.error("getNewUUID() requires a callback function with err, data parameters");
-        return;
+        throw "getNewUUID() requires a callback function with err, data parameters";
     }
 
     ::_uuidcb <- cb;
-    http.get("https://www.uuidgenerator.net/").sendasync(utilities._extractUUID);
+    http.get("https://www.uuidgenerator.net/api/version1").sendasync(utilities._extractUUID);
```

Comment:
Docs should also explicitly say this method is for use on agent only.
Should this check to make sure we are not on the device before calling an http.get?

Reply:
This is about to change, anyway